### DISCUSSION TITLE
Is there some way in the UI to view table indexes?

### DISCUSSION THREAD

**Discussion by 0radek** [supabase/supabase#4212] [Questions] [answered]
I used the SQL editor to create an index. Is there a way to quickly view indexes without running `select *
from pg_indexes` in the editor?

**Comment by GaryAustin1** [comment, marked as answer]
No.  I think the UI is very lacking around indexes.
I use something like DataGrip (there are many other tools) to manage the database, especially to quickly see the current table structures, triggers and indexes.

**Reply by 0radek** [reply]
Thanks.

**Reply by kylerummens** [reply]
While this works for viewing indexes, I get the error "must be owner of the table example_table" when I try to add/modify indexes

**Reply by GaryAustin1** [reply]
I did 
```
alter table "name"
    owner to postgres;
```
from the sql editor to solve that.
Not sure it is the best way, but have not had any issues.
